docs/collector.logical_disk.md

```diff
@@ -24,6 +24,8 @@ If given, a disk needs to *not* match the blacklist regexp in order for the corr
 Name | Description | Type | Labels
 -----|-------------|------|-------
 `requests_queued` | Number of requests outstanding on the disk at the time the performance data is collected | gauge | `volume`
+`avg_read_requests_queued` | Average number of read requests that were queued for the selected disk during the sample interval | counter | `volume`
+`avg_write_requests_queued` | Average number of write requests that were queued for the selected disk during the sample interval | counter | `volume`
```

Comment:
Should these two metrics be gauges rather than counters? The metric descriptions imply that the value is not incrementing.

Reply:
Yeah it should be gauge. I have made new push with the correct metric description.

Reply:
Thanks for that :+1:

There's one more locations for the `counter -> gauge` change, I've added another comment.

Reply:
Done, I have updated that as well. Thanks for reviewing.